lib/components/ContentPanel/ContentPanel.module.scss

```diff
@@ -0,0 +1,134 @@
+@import '../../common/color.controls';
+@import '../../common/constants';
+@import '../../common/mixins';
+
+$content-pane-width: 320px;
+$title-font-size: 20px;
+$close-icon-height: $grid-size * 8;
+$title-height: $gutter-xsmall+$title-font-size;
+$box-shadow-panel: 0 25.6px 57.6px 0 rgba(0, 0, 0, .12);
+.content-panel {
+    width: $content-pane-width;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    // border: 1px solid; // for the purpose
+    margin-left: calc(100% - #{$content-pane-width});
+    padding: $gutter-normal;
+    box-shadow: $box-shadow-panel;
+
+    .close-button {
+        align-self: flex-end;
+        margin: -#{$close-icon-height / 2};
+        height: $close-icon-height;
+    }
+
+    .title {
+        font-size: $title-font-size;
+        height: $title-height;
+        line-height: $title-height;
+        margin-bottom: $gutter-big;
+    }
+
+    .content {
+        height: calc(100% - #{$title-font-size + $close-icon-height});
+        overflow-y: auto;
+        overflow-x: hidden;
+        width: $content-pane-width - $gutter-normal -1;
+        -webkit-animation-name: slide-fade-in;
+        animation-name: slide-fade-in;
+        -webkit-animation-duration: .5s;
+        animation-duration: .5s;
+        -webkit-animation-timing-function: cubic-bezier(.8, 0, .2, 1);
+        animation-timing-function: cubic-bezier(.8, 0, .2, 1);
+        -webkit-animation-fill-mode: backwards;
+        animation-fill-mode: backwards;
+        -webkit-animation-delay: 80ms;
+        animation-delay: 80ms;
+    }
+
+    .separator {
+        border-top-width: 1px;
+        border-top-style: solid;
+        margin-left: -$gutter-normal;
+        width: $content-pane-width;
+    }
+
+    .actions {
+        padding-top: $gutter-normal;
+
+        .btn {
+            margin: 0;
+        }
+
+        .btn:first-child {
+            margin-right: $gutter-xsmall;
+        }
+    }
+
+    @include themify {
+        background-color: themed('color-bg-searchbar');
+
+        .separator {
+            border-top-color: themed('color-border-navbar-separator');
+        }
+    }
+
+    @include rtl {
+        .content-panel {
+            margin-left: 0;
+
+            .separator {
+                margin-left: 0;
+                margin-right: -$gutter-normal;
+            }
+
+            .actions {
+                .btn.btn-default {
+                    margin-left: 0;
+                    margin-right: $gutter-xsmall;
+                }
+            }
+        }
+    }
+
+    &.sm {
```

Comment:
nit: since .sm and .md have the same styles, can merge the two

Reply:
they don't have the same style